lib/puppet/type/foreman_host.rb

```diff
@@ -48,4 +48,8 @@
 
     defaultto 500
   end
+
+  autorequire(:class) do
```

Comment:
Sadly, you can't autorequire classes. I'll see if I can patch up the existing types so you can copy that pattern.

Reply:
That would be much appreciated

Reply:
https://github.com/theforeman/puppet-foreman/pull/966

Reply:
Just to be sure I got this right before I start making changes. I should place a named anchor into `manifests/init.pp` and then autorequire it from here?